tools/jdbc/src/test/java/org/duckdb/test/TestDuckDBJDBC.java

```diff
@@ -533,12 +533,9 @@ public static void test_throw_wrong_datatype() throws Exception {
 
 		rs = stmt.executeQuery("SELECT * FROM t");
 		rs.next();
-
-		try {
-			rs.getTimestamp(2);
-			fail();
-		} catch (IllegalArgumentException e) {
-		}
+		
+		//	This succeeds now,
+		rs.getTimestamp(2);
```

Comment:
We should probably keep this test as one that tests for the exception? So maybe switch to another data type (bool maybe?)